### PR TITLE
Document plugin removal steps

### PR DESCRIPTION
**Pull request by jeabakker**
fixes: #9419
ref: #10045

### CODE REVIEW

File: docs/contribute/core/move_plugin.rst [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+	git subtree split
+
+and the guide calls for
+
+.. code:: sh
+
+	git filter-branch --prune-empty --subdirectory-filter
+
+This will leave a difference in the commits which GiHub is unable to resolve. In that case you'll have to force push to changes to the Elgg repository.
```
Comment (mrclay) [inline docs/contribute/core/move_plugin.rst]: You mean force push to the new plugin repo, right?

Comment (jdalsem) [inline docs/contribute/core/move_plugin.rst]: GitHub

Comment (jdalsem) [inline docs/contribute/core/move_plugin.rst]: **the** changes

**Comment by jeabakker** [comment]
docs updated

**Review by jdalsem — APPROVED** [review]

**Pull request merged**